coolkit/dependencies/dependency.py

```diff
@@ -2,22 +2,21 @@
 import subprocess
 import sys
 
+import dependencies
```

Comment:
dont import this.
keep this script independent of this.
this wont work when called from outside